web.go

```diff
@@ -83,6 +84,22 @@ func setupRouter() *gin.Engine {
 		}),
 	))
 
+	if secureRedirect {
+		r.Use(secure.Secure(secure.Options{
+			AllowedHosts:          []string{},
+			SSLRedirect:           true,
+			SSLHost:               "",
+			SSLProxyHeaders:       map[string]string{"X-Forwarded-Proto": "https"},
+			STSSeconds:            315360000,
```

Comment:
We should *not* be setting STS requirements. It can be normal best-practice for very strongly secured websites, but if you set STS on a domain, the browser is going to cache that in an almost unclearable way (it's a **real PITA** to undo) and might make getting back to the device very difficult if there's some cert setup failure.